packages/patternfly-4/react-core/src/components/ApplicationLauncher/InternalApplicationLauncherText.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+
+export interface ApplicationLauncherTextProps extends React.HTMLProps<HTMLSpanElement> {
+  /** content rendered inside the text container */
+  children: React.ReactNode;
+  /** Additional classes added to the text container */
+  className?: string;
+}
+
+export const ApplicationLauncherText: React.FunctionComponent<ApplicationLauncherTextProps> = ({
+  className = '',
+  children,
+  ...props
+}: ApplicationLauncherTextProps) => (
+  <span className={css('pf-c-app-launcher__menu-item-text', className)} {...props}>
```

Comment:
Needs to use `style` object `css(styles.appLauncherMenuItemText)`

Reply:
This class does not actually exist (yet) but it was applied in the core example. @mcoker , should I keep it around or remove it for now?

Reply:
@jschuler is there a reason we shouldn't keep it? Since it isn't being used, I can always just remove it and add it back when it's needed.

Reply:
i think it's fine to keep it around for now